### PR TITLE
refactor/PSD-2519:rspec_and_coverage_updates

### PR DESCRIPTION
**Pull request by Nashar101**
Description
Increased spec coverage for helpers

### CODE REVIEW

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1,16 @@
+c
```
Comment (alan-at-work) [inline .byebug_history:1]: Need to remove this file as it shouldn't really get committed :)

File: app/helpers/notifications_helper.rb [hunk 2]
```diff
@@ -4,10 +4,10 @@ def show_edit_link?
   end
 
   def collaborator_access(collaborator)
-    case collaborator
```
Comment (alan-at-work) [inline app/helpers/notifications_helper.rb:7]: Not sure this is the best way to do this, you should not need to convert it to a string?

If the values are symbols (how it was done), you should use symbols directly for comparison. Here's how you can do it:

```ruby
def collaborator_access(collaborator)
  case collaborator
  when Collaboration::Access::Edit
    "Edit"
  when Collaboration::Access::ReadOnly
    "View"
  else
    "Unknown Access Level"
  end
end
```

If collaborator is a string and the constants are symbols, you can use to_sym to convert the string to a symbol for comparison:

```ruby
def collaborator_access(collaborator)
  case collaborator.to_sym
  when Collaboration::Access::Edit
    "Edit"
  when Collaboration::Access::ReadOnly
    "View"
  else
    "Unknown Access Level"
  end
end
```

File: spec/features/add_and_remove_a_business_spec.rb [hunk 3]
```diff
@@ -384,7 +384,6 @@ def record_a_corrective_action
 
     within_fieldset "Which business is responsible?" do
       choose "Trading name (Retailer)"
-      # TODO: add test here once business selection is possible
```
Comment (alan-at-work) [inline spec/features/add_and_remove_a_business_spec.rb:387]: This has not been finished, so why remove the TODO?

File: spec/features/assert_formatted_product_and_notification_methods_spec.rb [hunk 4]
```diff
@@ -252,7 +252,6 @@
 
       within_fieldset "Which business is responsible?" do
         choose "Trading name (Retailer)"
-        # TODO: add test here once business selection is possible
```
Comment (alan-at-work) [inline spec/features/assert_formatted_product_and_notification_methods_spec.rb:255]: This has not been finished, so why remove the TODO?

File: spec/features/visit_risk_assessment_from_notification_task_list_page_spec.rb [hunk 5]
```diff
@@ -243,7 +243,6 @@
 
     within_fieldset "Which business is responsible?" do
       choose "Trading name (Retailer)"
-      # TODO: add test here once business selection is possible
```
Comment (alan-at-work) [inline spec/features/visit_risk_assessment_from_notification_task_list_page_spec.rb:246]: This has not been finished, so why remove the TODO?

File: spec/features/add_and_remove_a_business_spec.rb (hunk 3 above)
Comment (Nashar101) [reply spec/features/add_and_remove_a_business_spec.rb:387]: It is done the business selection is the choose "Trading name" there

File: spec/features/assert_formatted_product_and_notification_methods_spec.rb (hunk 4 above)
Comment (Nashar101) [reply spec/features/assert_formatted_product_and_notification_methods_spec.rb:255]: It is done the business selection is the choose "Trading name" there

File: spec/features/visit_risk_assessment_from_notification_task_list_page_spec.rb (hunk 5 above)
Comment (Nashar101) [reply spec/features/visit_risk_assessment_from_notification_task_list_page_spec.rb:246]: It is done the business selection is the choose "Trading name" there

File: .byebug_history (hunk 1 above)
Comment (Nashar101) [reply .byebug_history:1]: I did not commit this, will remove

File: app/helpers/notifications_helper.rb (hunk 2 above)
Comment (alan-at-work) [reply app/helpers/notifications_helper.rb:7]: fair enough 👍

File: spec/features/add_and_remove_a_business_spec.rb (hunk 3 above)
Comment (alan-at-work) [reply spec/features/add_and_remove_a_business_spec.rb:387]: fair enough 👍

File: spec/features/assert_formatted_product_and_notification_methods_spec.rb (hunk 4 above)
Comment (alan-at-work) [reply spec/features/assert_formatted_product_and_notification_methods_spec.rb:255]: fair enough 👍

File: spec/features/visit_risk_assessment_from_notification_task_list_page_spec.rb (hunk 5 above)
Comment (alan-at-work) [reply spec/features/visit_risk_assessment_from_notification_task_list_page_spec.rb:246]: fair enough 👍

**Review by alan-at-work — APPROVED** [review]

**Pull request merged**